app/components/library-item-form.js

```diff
@@ -0,0 +1,12 @@
+import Component from '@ember/component';
+import { action } from '@ember/object';
+
+export default class LibraryItemFormComponent extends Component {
+  // pass an action to override
+  handleClick = this.args?.handleClick;
```

Comment:
why did you need to do this?

Reply:
To overwrite the function in the inner component with the one passed in from the outer component.

Here:
https://github.com/zoltan-nz/library-app/blob/master/app/components/library-item-form.js